### PR TITLE
Add missing tab and notices for the Product Settings block

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds missing tab and notices for the Product Settings block

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/settings.html

### CODE REVIEW

File: src/catalog/settings.md [hunk 1]
```diff
@@ -15,11 +15,14 @@ _Product Settings_
...
 |[Customizable Options]({% link catalog/settings-advanced-custom-options.md %})|Adds customizable options to a product.|
 |[Product in Websites]({% link catalog/settings-basic-websites.md %})| Identifies each website where the product is available, according to the store hierarchy.|
 |[Design]({% link catalog/settings-advanced-design.md %})|Used to apply a different theme to the product page, change the column layout, determine where product options appear, and enter custom XML code.|
 |[Gift options]({% link catalog/product-gift-options.md %})|Used to enable or disable a gift message option during checkout at the product level.|
-|[Downloadable Information]({% link catalog/product-downloadable-information.md %})|Used to generate the links to downloadable products and samples.|
\ No newline at end of file
+|{:.b2b-only}[Product In Shared Catalogs]({% link catalog/catalog-shared.md %})| Gives the ability to maintain gated shared catalogs with custom pricing for different companies and provides access to two types of shares catalogs with different pricing structures.|
```
Comment (leslietilling) [inline src/catalog/settings.md:27]: Unfortunately, `{:.b2b-only}` does not work in tables

```suggestion
|<span class="b2b-only">[Product In Shared Catalogs]({% link catalog/catalog-shared.md %})</span>| Enables the ability to maintain shared catalogs with custom pricing for different companies, with access to two types of shared catalogs using different pricing structures.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
syntax fix and minor edit

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**